bitnami/cassandra/README.md

```diff
@@ -92,8 +92,9 @@ The following tables lists the configurable parameters of the cassandra chart an
 | `cluster.rack`                       | Rack name                                                                                                                                                 | `rack1`                                                      |
 | `cluster.enableRPC`                  | Enable Thrift RPC endpoint                                                                                                                                | `true`                                                       |
 | `cluster.enableUDF'                  | Enable CASSANDRA_ENABLE_USER_DEFINED_FUNCTIONS                                                                                                            | `false`                                                      |
-| `cluster.pdbEnabled`                 | Enable the creation of a Pod Disruption Budget to ensure a minimun number of pods are running.                                                            |               `true`                                                      |
-| `cluster.minimumAvailable`           | Minimum number of instances that must be available in the cluster (used of PodDisruptionBudget) Needs `cluster.pdbEnabled=true`                           | `1`                                                          |
+| `cluster.pdbEnabled`                 | Enable the creation of a Pod Disruption Budget to ensure a minimun number of pods are running.                                                            | `true`                                                       |
+| `cluster.minAvailable`               | Minimum number of instances that must be available in the cluster (used of PodDisruptionBudget) Needs `cluster.pdbEnabled=true`                           | `nil`                                                        |
+| `cluster.maxUnavailable`             | Maximum number of instances that can be unavailable in the cluster (used of PodDisruptionBudget) Needs `cluster.pdbEnabled=true`                          | `nil`                                                        |
```

Comment:
For coherence , this should be `cluster.maximumUnavailable`

Reply:
minAvailable and maxUnavailable are used in other charts + minAvailable is currently used in go code. This is documentation change for minAvailable and for coherence new variable should be maxUnavailable, correct me if I'm wrong

Reply:
OK, I have checked what you mentioned about other charts.
I agree with keeping minAvailable and maxUnavailable.
Please add a note to readme mentioning the renaming of minimunAvailable, and bump the major version of the chart instead of the minor.
Thanks